validator/lib/isTile.js

```diff
@@ -1,29 +1,43 @@
 'use strict';
 const path = require('path');
 const Cesium = require('cesium');
+const defaultValue = Cesium.defaultValue;
 const defined = Cesium.defined;
 module.exports = isTile;
 
+var emptyArray = [];
+
 /**
  * Checks whether the given file path is a tile file path.
  *
  * @param {String} filePath The file path.
- * @param {String} version The tileset version
+ * @param {Object} [options] Options object
  * @returns {Boolean} True if the file path is a tile file path, false if not.
  */
-function isTile(filePath, version) {
+function isTile(filePath, options) {
     var extension = path.extname(filePath);
-    if (defined(version)) {
-        if (version === '1.0') {
+    if (defined(options)) {
+        var version = options.version;
+        var tileset = options.tileset;
+        var extensionsUsed = defaultValue(tileset.extensionsUsed, emptyArray);
+        var hasContentGltfExtension = extensionsUsed.indexOf('3DTILES_content_gltf') > -1;
```

Comment:
This marks a tileset as valid if `3DTILES_content_gltf` is in `extensionsUsed` and not `extensionsRequired`. I'm assuming any tileset that needs this extension will need to add it to the `extensionsRequired`, correct?

Reply:
Hm this makes me wonder...

[3DTILES_content_gltf](https://github.com/CesiumGS/3d-tiles/tree/3DTILES_content_gltf/extensions/3DTILES_content_gltf) says it needs to go in `extensionsRequired` but now I'm thinking about the interaction with [3DTILES_layers](https://github.com/CesiumGS/3d-tiles/pull/432) and how someone might create a tileset with both b3dm and glTF layers so that the b3dm layer can be a fallback if the client doesn't support `3DTILES_content_gltf`. That's a possible use case for allowing `3DTILES_content_gltf` to be optional.

Reply:
I left a comment in https://github.com/CesiumGS/3d-tiles/pull/427#issuecomment-715598362.

Reply:
For this area of the code it's ok to just check `extensionsUsed` since the extension will always go in `extensionsUsed` whether it's required or not.

I pushed some changes that do more thorough checking of the behavior of `3DTILES_content_gltf`. Also went through and fixed some broken unit tests and eslint errors.